monitor.go

```diff
@@ -192,20 +198,30 @@ func NewMonitor(opts ...MonitorOption) *Monitor {
 }
 
 func (monitor *Monitor) monitor() {
+	changesEvents := make([]*MonitorEvent, 0)
```

Comment:
Write `var changesEvents []*MonitorEvent` here. `make(…, 0)` is equivalent to just declaring a variable.

Reply:
changed to declaration. Thank you